package.json

```diff
@@ -8,6 +8,7 @@
     "build:watch": "npm run build -- --watch",
     "test": "mocha \"./src/**/*.test.js\" --require babel-core/register",
     "test:watch": "npm run test -- --watch",
+    "flow": "flow; test $? -eq 0 -o $? -eq 2",
```

Comment:
No idea what the whole `test $? -eq 0 -o $? -eq 2` thing does, just straight copied that from the flow website "Getting started" guide